// Copyright 2022 ByteDance and/or its affiliates.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";

package monolith.native_training.data.config;


message TaskLabelConf {
  repeated int32 pos_actions = 1;
  repeated int32 neg_actions = 2;
  optional float sample_rate = 3 [default = 1.0];
}

message LabelConf {
  repeated TaskLabelConf conf = 1;
}